test/functional/wallet_bumpfee.py

```diff
@@ -283,10 +283,10 @@ def test_single_output(self):
         tx = wallet.sendall(recipients=[wallet.getnewaddress()], fee_rate=2, options={"inputs": [utxos[0]]})
 
         # Reduce the only output with a crazy high feerate, should fail as the output would be dust
```

Comment:
We might want to change this comment to
"Change the output index to the only output with a crazy high feerate, should fail as the output would be dust"

Reply:
done, thx